src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -136,6 +161,16 @@ protected Message getInboundQueueMessage() {
         return waitFor(() -> jmsTemplate.receive(meshInboundQueueName));
```

Comment:
Should we remove this method as we don't use it anymore. What do you think?

Reply:
Oh yes, we should remove this - done

Reply:
Perfect. Thanks!